packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -425,7 +425,8 @@ export default mixins(
 
 					saveAs(blob, workflowName + '.json');
 				} else if (key === 'workflow-save') {
-					this.saveCurrentWorkflow(undefined);
+					await this.saveCurrentWorkflow(undefined);
+					this.$store.dispatch('settings/fetchPromptsData');
```

Comment:
same question here... what should happen if a workflow is not saved?

Reply:
Fixed